src/components/ModalWindow/ModalWindow.jsx

```diff
@@ -0,0 +1,183 @@
+import React, {useState, useEffect} from 'react';
+import {useSelector} from 'react-redux';
+import {NavLink} from 'react-router-dom';
+import Loader from '../Loader';
+
+import './modalwindow.css';
+
+export const ModalWindow = ({onClose}) => {
```

Comment:
You need to make ModalWindow component generic. And create PersonModalWindow component, which will use ModalWindow component